docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

```diff
@@ -45,6 +45,7 @@ Some stable features are enabled by default. You can disable a stable feature by
 | `autoMigrateGraphPanels`          | Replace the angular graph panel with timeseries                                 |
 | `datasourceLogger`                | Logs all datasource requests                                                    |
 | `accessControlOnCall`             | Access control primitives for OnCall                                            |
+| `alertingNoNormalState`           | Stop maintaining state of alerts that are not firing                            |
```

Comment:
Can we artificially assume this state instead of putting this behind a feature flag then? As in, could we assume (and insert at runtime) a new state that would assume normal when a previous state its state is missing.

Reply:
we can artificially assume a state for a rule, but we cannot assume a state for each rule's instances when Grafana starts. The normal state will be available only after the first evaluation once Grafana starts, which can be in 1s or 1h or more.

Reply:
I mean this from a presentation point of view. From state management pov we already do this (i.e. artificially assume a new instance to be in normal state. The difference is in presentation: currently, we show instances with normal state in UI, with this feature flag enabled, we are not able to show until the first evaluation after the restart. Therefore, I decided to not show it at all because it could be confusing (especially in environments when Grafana can be restarted at any time, like K8s)